install/kubernetes/cilium/files/cilium-envoy/configmap/bootstrap-config.json

```diff
@@ -54,6 +54,12 @@
                   ],
                   "internal_address_config": {
                     "cidr_ranges": [
+                      {{- if and .Values.ipv4.enabled .Values.ipv6.enabled }}
+                      {
+                        "address_prefix": "::1",
+                        "prefix_len": 128
+                      },
+                      {{- end }}
                       {
                         "address_prefix": "10.0.0.0",
```

Comment:
Should these be configured only if IPv4 is enabled?

Reply:
good point

Reply:
I would still structure this a bit simpler, e.g., 
```
if ipv6
  <IPV6 address config>
end
if ipv4.enabled
  <IPv4 address config>
end
```
This way there is no duplication of the IPv6 address config.

Reply:
unlike yaml, we need to handle the comma symbol in array, so it's why ipv4 and ipv4 checks are nested.